### DISCUSSION TITLE
【2023.10.12】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#1149] [meeting]
#### Meeting replay: [Bilibili](https://www.bilibili.com/video/BV1Du4y1W7Zd/?spm_id_from=333.999.0.0)

**Conference link：**
> **Note**: Every two weeks at 3:00 p.m
https://meeting.tencent.com/dm/y6D9zgQbcXmF 📱#Tencent Conference：333-9699-4866

This biweekly will be a long awaited mega-merge and the beginning of our `OpenIM:3.4.0` merge

**Outline of the meeting:**
- [User case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md) and [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**If you have a topic to discuss, please comment below.**

**Comment by IRONICBo** [comment]
- openkf release :)

Ref: https://github.com/openimsdk/openkf
Deploy link: https://www.bilibili.com/video/BV1494y1Y7Et

**Comment by plum330** [comment]
1. Now that the service discovery in the code is implemented through zk, how to get all the connections if server is deployed in k8s?
2. After the message is transferred to msg-transfer, if the batch insert redis fails, mongo does not save, and will not push, but the sender sees that it is successful (the sender thinks that the message is sent to kafka is successful), how to deal with this situation? Because I see a logical error in the code directly return. May I ask how openim ensures that the message must be reached? In the code did not see the logic to ensure that the message will reach.

**Reply by cubxxw** [reply]
post processing, 
+ todo: seq manager

**Comment by cubxxw** [comment]
How to use data migration from OpenIM V2 version to V3 version: https://github.com/openimsdk/open-im-server/blob/main/tools/data-conversion/info.md @withchao
